sandbox/playground.py

```diff
@@ -38,6 +38,7 @@
 from tac.agents.v1.base.dialogues import Dialogue
 from tac.agents.v1.examples.baseline import BaselineAgent
 from tac.agents.v1.examples.strategy import BaselineStrategy
+from tac.agents.v1.mail.messages import OEFAgentCfp
```

Comment:
OEFAgentCFP

Reply:
this needs to be replaced with new Message class

Reply:
Yes!